dyno/panda/dyno_panel.py

```diff
@@ -222,16 +222,28 @@ def writeFile():
 throttle_canvas.create_window(75, 45, window=throttle_slider)
 #---------------------
 
+def check_ack():
+    if not mcu_acknowledged:
+        data = driver.read()
+        if data:
+            mcu_acknowledged = True
+            print("ACK=SUCCESS;")
```

Comment:
The microcontroller is going to be printing this message. Not the lattepanda. We want to change this line to parse the input from the microcontroller